src/tests/commentsCounter.test.js

```diff
@@ -0,0 +1,39 @@
+import commentsCounter from '../modules/commentsCounter.js';
```

Comment:
**Changes Required**
- [ ] There seem to be an issue importing your `commentsCounter` class which is making the test fail. Let us try to fix this. Below is the error I get when I run `npm test` on the `development` branch of your project.
![failing-test](https://user-images.githubusercontent.com/7220167/132782574-9e056f81-d621-44a6-98cd-68c3db6efd57.png)

Reply:
We cloned a new repo and immediately after npm install the tests we're working fine. We're unable to find any error.